src/components/organisms/Settings/Workspace/Asset/hooks.ts

```diff
@@ -8,6 +8,15 @@ import {
 } from "@reearth/gql";
 import { useLocalState } from "@reearth/state";
 
+export type Asset = {
```

Comment:
Is this type of definition used? If not, please delete it. 👍

Reply:
thanks